### PR TITLE
update to hyper 1

### PR DESCRIPTION
**Pull request by tottoto**
Updates the hyper example to use hyper 1.

### CODE REVIEW

File: examples/hyper.rs [hunk 1]
```diff
@@ -72,16 +76,70 @@ async fn main() {
...
+}
+
+mod body {
+    use std::convert::Infallible;
+    use std::pin::Pin;
+    use std::task::{Context, Poll};
+
+    use hyper::body::{Body as HttpBody, Bytes, Frame};
+
+    pub enum Body {
```
Comment (ibraheemdev) [inline examples/hyper.rs:110]: Can we use [`hyper_body_util::Full<Bytes>`](https://docs.rs/http-body-util/0.1.2/http_body_util/struct.Full.html) for the body type instead? This code is a little distracting from the main router code.

Comment (tottoto) [reply examples/hyper.rs:110]: > Can we use [hyper_body_util::Full<Bytes>](https://docs.rs/http-body-util/0.1.2/http_body_util/struct.Full.html) for the body type instead?

Yes. The easy way is just using `http_body_util::Full<Bytes>`, to describe empty body, using `Full::default` constructor.

Comment (tottoto) [reply examples/hyper.rs]: Updated.

**Comment by ibraheemdev** [comment]
Thanks!

**Pull request merged**